lib/fluent/plugin/out_aws-elasticsearch-service.rb

```diff
@@ -20,6 +20,7 @@ class AwsElasticsearchServiceOutput < ElasticsearchOutput
       config_param :ecs_container_credentials_relative_uri, :string, :default => nil #Set with AWS_CONTAINER_CREDENTIALS_RELATIVE_URI environment variable value
       config_param :assume_role_session_name, :string, :default => "fluentd"
       config_param :assume_role_web_identity_token_file, :string, :default => nil
+      config_param :sts_credentials_region, :string, :default => ""
```

Comment:
Nit: Also add this in spec?
https://github.com/atomita/fluent-plugin-aws-elasticsearch-service/blob/256c739e0a4494775052035954cba0c9d6966f81/spec/lib/fluent/plugin/out_aws-elasticsearch-service_spec.rb

Reply:
Refactored the code for better testability, but held off on updating tests since it looks like tests are broken even on master.